### PR TITLE
JAMES-2147 Add mentions of JIRA labels on the website

### PR DESCRIPTION
**Pull request by chibenwa**
capture d ecran de 2017-09-22 17-22-43

### CODE REVIEW

**Comment by mbaechler** [comment]
I don't want to restrict contribution to "the code".
We'll have a lot of non-dev users and I want to welcome them as well.
We should split contributions into several types : 
- help to write / fix some doc
- help to package things
- help on marketing stuff
- help on contributing to code
...

**Comment by chibenwa** [comment]
And here we go with the updated version!

capture d ecran de 2017-09-28 11-53-23

**Comment by aduprat** [comment]
Also, we can speak about sharing customers own mailets, in order to build a mailet store.

**Comment by chibenwa** [comment]
> Also, we can speak about sharing customers own mailets, in order to build a mailet store.

Cool idea. Maybe we can all a documentation page on the website for this and handle that in another ticket?

**Comment by chibenwa** [comment]
See https://issues.apache.org/jira/browse/JAMES-2165 @aduprat

**Comment by mbaechler** [comment]
Ok but italic is weirdly rendered, could we fix that ?

**Comment by chibenwa** [comment]
Using blod?

```
<b>Something important</b>

**Comment by chibenwa** [comment]
capture d ecran de 2017-10-02 18-23-32

File: src/homepage/index.html [hunk 1]
```diff
@@ -175,18 +175,20 @@ <h2 class="big-h2"><span class="fa fa-wrench"></span> <span>James is a <b>living
...
                   You can also come and interact with us on <a href="https://github.com/apache/james-project" class="comm" alt="James on GitHub">GitHub</a>.<br/>
-                  Here are some more instructions to <a href="/contribute.html" class="comm" alt="Contributing to James: guidelines">get starting contributing</a>.
                 </span>
               </li>
               <li class="post-template center-icon">
                 <a href="https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20newbie%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC" alt="contribute to the code"><span class="icon fa-cogs"></span></a>
                 <span class="details">
-                  <b>Contribute to the code:</b>
-                  <br/>Get started with code contribution! Search out the
+                  <b>Get started and contribute to this project:</b>
```
Comment (strokyl) [inline src/homepage/index.html]: Next time please add breakline otherwise it's a nightmare to review that one github !

**Pull request closed without merging**